web/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:12-alpine
+
+WORKDIR /home/app
+COPY . .
+
+RUN addgroup -S appgroup
+RUN adduser -S -D -h /home/app app appgroup
+RUN chown -R app:appgroup /home/app
```

Comment:
Do we need these?

Reply:
Open quetsion, do we even need a Dockerfile?

How about having a named volumes for node modules and running yarn on launch?

Your approach works but it means that we will have to re-build the docker whenever we change a library

Reply:
For development, we don't need them.